dummy/modowners_test.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"bytes"
+	"io/fs"
+	"log"
+	"strings"
+	"testing"
+	"testing/fstest"
+)
+
+func TestCommonElement(t *testing.T) {
+	for _, test := range []struct {
+		A      []string
+		B      []string
+		Result bool
+	}{
```

Comment:
A slightly more idiomatic way of doing this can also look like

```
func TestCommonElement(t *testing.T) {
  tests := []struct {
    testDesc string
    A string
    B string
    ExpectedResult string
  }{
    {
        testDesc: "hello world test",
        A: "etc",
        B: "etc",
        ExpectedResult string
    }
  }
  for _, test := range tests {
    t.Run(test.testDesc, func(t *testing.T)) {
        //add functional code and assertions
    }
  }

```

Reply:
ah this is cool - thank you!